docs/slides.md

```diff
@@ -38,7 +38,7 @@ date: Thursday, November 24th, 2022
 
 # Example FTP Workflow
 
-![FTP Workflow](docs/ftp-workflow.png)
```

Comment:
This should stay as-is, but the other change is okay.

Reply:
I've reverted the change!